lib/OpenQA/Jobs/Constants.pm

```diff
@@ -97,6 +97,8 @@ use constant {
 # define results for particular job modules
 use constant MODULE_RESULTS => (CANCELLED, FAILED, NONE, PASSED, RUNNING, SKIPPED, SOFTFAILED);
 
+use constant COMMON_RESULT_FILES => ('vars.json', 'autoinst-log.txt', 'worker-log.txt');
```

Comment:
Since all kinds of variables have a documenting comment it would be nice to have one here as well for consistency.

Reply:
done